ci-operator/config/operator-framework/operator-metering/operator-framework-operator-metering-master.yaml

```diff
@@ -34,7 +34,7 @@ build_root:
     context_dir: .
 tests:
 - as: unit
-  commands: make unit
+  commands: make unit TEST_OUTPUT_PATH=/tmp/artifacts
```

Comment:
shouldn't this be TEST_OUTPUT_DIR? 

https://github.com/operator-framework/operator-metering/pull/605/files#diff-3013a7c42949980d2af09a722fb3f5beR7

Reply:
Oops yes